freeipa/src/main/java/com/sequenceiq/freeipa/service/stack/FreeIpaDescribeService.java

```diff
@@ -49,6 +54,9 @@ public DescribeFreeIpaResponse describe(String environmentCrn, String accountId)
 
     public List<DescribeFreeIpaResponse> describeAll(String environmentCrn, String accountId) {
         List<Stack> stacks = stackService.findMultipleByEnvironmentCrnAndAccountIdEvenIfTerminatedWithList(environmentCrn, accountId);
+        if (!entitlementService.isFreeIpaRebuildEnabled(accountId)) {
```

Comment:
Could you move this up one line? No need to fetch anything from the db if there is no entitlement

Reply:
Good point. Thanks.